src/main.ts

```diff
@@ -5,11 +5,20 @@ import { poTokenGenerate, type TokenMinter } from "./lib/jobs/potoken.ts";
 import { USER_AGENT } from "bgutils";
 import { retry } from "@std/async";
 import type { HonoVariables } from "./lib/types/HonoVariables.ts";
+import { parseArgs } from "@std/cli/parse-args";
 
 import { parseConfig } from "./lib/helpers/config.ts";
 const config = await parseConfig();
 import { Metrics } from "./lib/helpers/metrics.ts";
 
+const args = parseArgs(Deno.args);
```

Comment:
Isn't that going to create issues if we start to introduce an argument system: https://github.com/iv-org/invidious-companion/issues/33?